modules/lvms-limitations-to-configure-size-of-devices.adoc

```diff
@@ -23,36 +23,54 @@ ifdef::microshift[]
 endif::microshift[]
 
 ifndef::microshift[]
-.Size limits for different architectures using the default PE and LE size
-[cols="1,1,1,1,1", width="100%", options="header"]
+.Tested configuration
+[cols="1,1", width="100%", options="header"]
 |====
-|Architecture
-|RHEL 6
-|RHEL 7
-|RHEL 8
-|RHEL 9
 
-|32-bit
-|16 TB
-|-
-|-
-|-
+|Parameter
+|Value
 
-|64-bit
+|Chunk size
+|128 KiB
 
-|8 EB ^[1]^
+|Maximum volume size
+|32 TiB
 
-100 TB ^[2]^
-|8 EB ^[1]^
+|====
+
+.Theoretical size limits for static configuration
+[cols="1,1,1", width="100%", options="header"]
+|====
+
+|Parameter
+|Minimum value
+|Maximum value
+
+|Chunk size
+|64 KiB
+|1 GiB
+
+|Volume size
+|Minimum size of the underlying {op-system-first} system
+|Maximum size of the underlying RHCOS system
```

Comment:
```suggestion
|Maximum size of the underlying {op-system} system
```

Worth checking the MShift build to make sure this attribute renders.

Reply:
Looks like these tables don't apply to MShift, so using the attribute should not cause issues in the core docs.

Reply:
Updated as per the suggestion.